pkg/utils/utils.go

```diff
@@ -169,19 +176,30 @@ func getPodDefinition(node string, namespace string, image string) *corev1.Pod {
 			NodeName:                      node,
 			PriorityClassName:             "openshift-user-critical",
 			RestartPolicy:                 corev1.RestartPolicyNever,
-			TerminationGracePeriodSeconds: ptr.To[int64](1),
+			TerminationGracePeriodSeconds: ptr.To(int64(1)),
 			Tolerations: []corev1.Toleration{
 				{
 					Effect:            corev1.TaintEffectNoExecute,
 					Key:               "node.kubernetes.io/not-ready",
 					Operator:          corev1.TolerationOpExists,
-					TolerationSeconds: ptr.To[int64](300),
+					TolerationSeconds: ptr.To(tolerationSeconds),
 				},
 				{
 					Effect:            corev1.TaintEffectNoExecute,
 					Key:               "node.kubernetes.io/unreachable",
 					Operator:          corev1.TolerationOpExists,
-					TolerationSeconds: ptr.To[int64](300),
+					TolerationSeconds: ptr.To(tolerationSeconds),
+				},
+			},
+			Volumes: []corev1.Volume{
+				{
+					Name: "host-root",
+					VolumeSource: corev1.VolumeSource{
+						HostPath: &corev1.HostPathVolumeSource{
+							Path: "/",
+							Type: ptr.To(corev1.HostPathType("Directory")), // Ensure the path is a directory
```

Comment:
You could use some constants for "Directory" here.

Reply:
i thought thats fine to be here im using it just here so maybe we can keep